pkg/platform/docker/util.go

```diff
@@ -51,3 +71,254 @@ func ParseDockerCmd(cmd string, kind utils.CmdType, idc Client) (string, string,
+		out, err := cmd.Output()
+		if err != nil {
+			utils.LogError(logger, err, "failed to get the current docker context")
+			return "", errors.New("failed to get alias")
+		}
+		dockerContext := strings.Split(strings.TrimSpace(string(out)), "\n")[0]
+		if len(dockerContext) == 0 {
+			utils.LogError(logger, nil, "failed to get the current docker context")
+			return "", errors.New("failed to get alias")
+		}
+		dockerContext = strings.Split(dockerContext, "\n")[0]
+		if dockerContext == "colima" {
+			logger.Info("Starting keploy in docker with colima context, as that is the current context.")
+			alias := "docker container run --name keploy-v2 " + envs + "-e BINARY_TO_DOCKER=true -p 36789:36789 -p 8096:8096 --privileged --pid=host" + ttyFlag + "-v " + os.Getenv("PWD") + ":" + os.Getenv("PWD") + " -w " + os.Getenv("PWD") + " -v /sys/fs/cgroup:/sys/fs/cgroup -v /sys/kernel/debug:/sys/kernel/debug -v /sys/fs/bpf:/sys/fs/bpf -v /var/run/docker.sock:/var/run/docker.sock -v " + os.Getenv("HOME") + "/.keploy-config:/root/.keploy-config -v " + os.Getenv("HOME") + "/.keploy:/root/.keploy --rm " + img
+			return alias, nil
+		}
+		// if default docker context is used
+		logger.Info("Starting keploy in docker with default context, as that is the current context.")
+		alias := "docker container run --name keploy-v2 " + envs + "-e BINARY_TO_DOCKER=true -p 36789:36789 -p 8096:8096 --privileged --pid=host" + ttyFlag + "-v " + os.Getenv("PWD") + ":" + os.Getenv("PWD") + " -w " + os.Getenv("PWD") + " -v /sys/fs/cgroup:/sys/fs/cgroup -v debugfs:/sys/kernel/debug:rw -v /sys/fs/bpf:/sys/fs/bpf -v /var/run/docker.sock:/var/run/docker.sock -v " + os.Getenv("HOME") + "/.keploy-config:/root/.keploy-config -v " + os.Getenv("HOME") + "/.keploy:/root/.keploy --rm " + img
+		return alias, nil
+	}
+	return "", errors.New("failed to get alias")
+}
+
+func addKeployNetwork(ctx context.Context, logger *zap.Logger, client Client) {
+
+	// Check if the 'keploy-network' network exists
+	networks, err := client.NetworkList(ctx, network.ListOptions{})
+	if err != nil {
+		logger.Debug("failed to list docker networks")
+		return
+	}
+
+	for _, network := range networks {
+		if network.Name == "keploy-network" {
+			logger.Debug("keploy network already exists")
+			return
+		}
+	}
+
+	// Create the 'keploy' network if it doesn't exist
+	_, err = client.NetworkCreate(ctx, "keploy-network", network.CreateOptions{})
+	if err != nil {
+		logger.Debug("failed to create keploy network")
+		return
+	}
+
+	logger.Debug("keploy network created")
+}
+
+func convertPathToUnixStyle(path string) string {
+	// Replace backslashes with forward slashes
+	unixPath := strings.Replace(path, "\\", "/", -1)
+	// Remove 'C:'
+	if len(unixPath) > 1 && unixPath[1] == ':' {
+		unixPath = unixPath[2:]
+	}
+	return unixPath
+}
+
+// ExtractPidNamespaceInode extracts the inode of the PID namespace of a given PID
+func ExtractPidNamespaceInode(pid int) (string, error) {
```

Comment:
`PidNameSpace` is called inode only, you can rename this function to `ExtractInodeByPid` or something.